content/pages/sponsorship.md

```diff
@@ -0,0 +1,157 @@
+To receive your A11yCat T-shirt and/or sticker, please follow the instructions on your donation confirmation email from Open Collective. These will be mailed out in monthly batches. Email us at “team [at] a11ytalks [dot] com” if you think we’ve missed you. We promise it wasn’t on purpose.
+
+<table class="centered">
+<thead class="inverse">
+<tr>
+ <th colspan="4">Join the A11yCat Club (Monthly Recurring)</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td></td>
+  <td>$5/month<br/>($60/year, $0.16/day)</td>
+  <td>$15/month<br/>($180/year, $0.50/day)</td>
+  <td>$25/month<br/>($300/year, $0.83/day)</td>
+</tr>
+<tr>
+  <td class="left">Name listed as a donor on website</td>
+  <td>X</td>
+  <td>x</td>
+  <td>x</td>
+</tr>
+<tr>
+  <td class="left">Name on slide at the beginning of<br/>each A11yTalk</td>
+  <td>X</td>
+  <td>x</td>
+  <td>x</td>
+</tr>
+<tr>
+  <td class="left">A11yCat sticker</td>
+  <td>1</td>
+  <td>2</td>
+  <td>3</td>
+</tr>
+<tr>
+  <td class="left">A11yCat T-shirt</td>
+  <td>X</td>
+  <td>x</td>
+  <td>x</td>
+</tr>
+</tbody>
+</table>
+
+<table class="centered">
+<thead class="inverse">
+<tr>
+ <th colspan="4">Join the A11yCat Club (One-Time Donation)</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td></td>
+  <td>$25 donation</td>
+  <td>$100 donation</td>
+  <td>$250 donation</td>
+</tr>
+<tr>
+  <td class="left">Name listed as a donor on website</td>
+  <td>X</td>
+  <td>x</td>
+  <td>x</td>
+</tr>
+<tr>
+  <td class="left">A11yCat sticker</td>
+  <td>1</td>
+  <td>2</td>
+  <td>3</td>
+</tr>
+<tr>
+  <td class="left">A11yCat T-shirt</td>
+  <td>X</td>
+  <td>x</td>
+  <td>x</td>
+</tr>
+</tbody>
+</table>
+
+### Organizations: Contact us
+
+We prefer to work with organizations who have a positive relationship with the web community and align with our [mission and values](/pages/mission). Please contact us if you are interested in supporting our project.
+
+* Email: team [at] a11ytalks [dot] com
+* Twitter: [@A11yTalks](https://www.twitter.com/a11ytalks)
+
+## Measuring our impact
+
+### A11yTalks YouTube stats
+
+*Last updated 11/19/2022*
+
+#### Lifetime
+<div class="cards">
+  <div class="card">
+    <p class="large">64</p>
+    <p>Archived videos</p>
+  </div>
+  <div class="card">
+    <p class="large">2.18k</p>
+    <p>YouTube subscribers</p>
+  </div>
+</div>
+
+#### Last 90 days
+
+<div class="cards">
+  <div class="card">
+    <p class="large">2.2k</p>
+    <p>Views</p>
+    <p>(27% more than previous 90 days)</p>
+  </div>
+  <div class="card">
+    <p class="large">288.3</p>
+    <p>Hours watched</p>
+    <p>(27% more than previous 90 days)</p>
+  </div>
+  <div class="card">
+    <p class="large">84+</p>
+    <p>New subscribers</p>
+    <p>(87% more than the previous 90 days)</p>
+  </div>
+</div>
+
+<h4 class="centered">Top 3 Videos</h4>
```

Comment:
I made the background black in the doc because these could be thumbnails. Use your discretion, but maybe we don't need the black background. 🤷